rest/blip_sync.go

```diff
@@ -964,6 +952,17 @@ func (bh *blipHandler) handleRev(rq *blip.Message) error {
 		minRevpos++
 	}
 
+	// Handle and pull out expiry
```

Comment:
Is there a reason this got moved down?  It's sitting in between the two attachment bits at the moment, which isn't ideal.

Reply:
The reason was just so I could group stuff that was the same vs stuff that was different between BLIP and REST. I'll move it back for now. Depends whether we want to try and make a function for the bit thats the same for both or not.